crates/forge_services/src/metadata.rs

```diff
@@ -0,0 +1,34 @@
+use std::fmt::Display;
+
+/// Holds Metadata about truncating, file paths, chars ranges.
+#[derive(Default)]
+pub struct Metadata(Vec<(&'static str, String)>);
+
+impl Display for Metadata {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        writeln!(f, "---")?;
+        for (k, v) in self.0.iter() {
+            writeln!(f, "{k}: {v}")?;
+        }
+        writeln!(f, "---")
+    }
+}
```

Comment:
Its simpler right now to simply use a `writeln` everywhere and work with strings. We can later migrate all the tool results to use a more structured output for which Metadata can be automatically derived.